### PR TITLE
feat: add jas1005

### LINKED ISSUE #445: Feature: Add @jas1005 as a Contributor
### Description

As part of Open-sauced's Intro to Open Source course, we have been instructed to add ourselves as contributors to this repository. 

### Suggested solution

Add myself as a contributor!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Getting Started Instructions

- [X] I agree to follow this project's Getting Started instructions

### PR DESCRIPTION
**Pull request by jas1005**
## Description

This PR adds @jas1005 as a contributor.

## What type of PR is this? (check all applicable)

- [X] 🤝 Add a contributor
- [ ] 📝 Documentation Update

## Related Issues

Closes #445 

### I've read through the [Getting Started](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=getting-started) section.

- [X] ✅ Yes
- [ ] ❌ Not yet

### Have you run `npm run contributors:generate` to generate your profile and the badge on the README?

- [X] ✅ Yes
- [ ] ❌ No

## Added to documentation?

- [X] 📜 README.md
- [ ] 🙅 no documentation needed

## Screenshot (Required for PR Review)

contributor_profile

### CODE REVIEW

**Comment by jas1005** [comment]
@open-sauced/docs, please see the comments section of my original pull request. I have updated the title of my pull request to be in accordance with the Open Sauced rules. Please advise me if there is anything else to be done before my code can be merged.

File: contributor_profile.png
Comment (adiati98) [reply contributor_profile.png:1]: You need to upload this image on your PR template for us to review, and no need to include it in your PR.

You can simply drag and drop the image to the PR template. :)

**Comment by adiati98** [comment]
Hey @jas1005,

We have merged a PR before yours. Please **[resolve the conflicts](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=merge-conflicts-in-the-guestbook-repository)** before we can review your PR.

Please let me know if you need any help. Thanks. 😊

File: contributor_profile.png
Comment (jas1005) [reply contributor_profile.png:1]: Thank you for the clarification! I have uploaded the image to the PR template. I will now resolve the merge conflicts and remove my profile confirmation from my commit.

**Comment by jas1005** [comment]
@adiati98 I think I've fixed all the issues in a way that's in accordance with contribution guidelines. You can see that I put my screenshot into the PR template above and undid my commit where I added a separate file. I had to manually fix the merge issues with me adding myself to the author list and all the other authors that added themselves to the author list between when I forked the repository. I think I've done this correctly though. Can you confirm this? If there's anything else I need to do, I would love your guidance on how to proceed. I spent a large fraction of today going down rabbit holes to figure out where I went wrong.

File: .all-contributorsrc
Comment (adiati98) [inline .all-contributorsrc:3]: We don't want to change this. Please change it back to the original.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hi @jas1005,
Thank you for resolving the merge conflicts!

I left a review here for you to fix.

Also, you need to run the command to generate your profile after you merged the conflicts. I don't see the badge increase here. 

Another thing is please provide a description of your PR in the PR template.

Let me know if you have questions or need help! ✨

File: .all-contributorsrc
Comment (jas1005) [reply .all-contributorsrc:3]: Hey, @adiati98. I don't know why this difference is showing up. When I'm on my main branch or my feature branch, I see the full 'TejasQ' on line 3. Can you advise on how I should locate the version of .all-contributorsrc that is giving this conflict?

**Comment by jas1005** [comment]
@adiati98, I think I figured it out. I hadn't pushed/pulled my changes to my fork in the right way. I think I've addressed all of the issues you mentioned in your most recent thread. Is there anything else I should do?

**Review by adiati98 — APPROVED** [review]
Well done and thank you for the fix, @jas1005! ✨ 

If you haven't, you can join our [Community](https://github.com/orgs/open-sauced/discussions/1).
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕🍴

git merge GIF

**Comment by jas1005** [comment]
@adiati98 @BekahHW, am I good to do `git merge` then? How exactly should I do that given my somewhat unique situation with my commits?

**Comment by adiati98** [comment]
> @adiati98 @BekahHW, am I good to do `git merge` then? How exactly should I do that given my somewhat unique situation with my commits?

You don't need to do anything. There will be a 2nd review, and one of the maintainers will merge your PR. 

So, how things go in open source is you create a PR, then the maintainer(s) of the project will review it. They might or might not ask you for fixes, and they will merge the PR after it meets the requirements. Hope this explanation helps. 🙂

**Comment by jas1005** [comment]
> > @adiati98 @BekahHW, am I good to do `git merge` then? How exactly should I do that given my somewhat unique situation with my commits?
> 
> You don't need to do anything. There will be a 2nd review, and one of the maintainers will merge your PR.
> 
> So, how things go in open source is you create a PR, then the maintainer(s) of the project will review it. They might or might not ask you for fixes, and they will merge the PR after it meets the requirements. Hope this explanation helps. 🙂

This helps clarify things. I had thought this was the case. But I wanted to make sure. Thanks!

**Review by BekahHW — APPROVED** [review]

**Comment by jas1005** [comment]
This is my first contribution to a repository that I have not owned. What's the next step in the process? I do not currently see my author entry on the open-sauced:main branch.

**Comment by BekahHW** [comment]
Hey @jas1005, @adiati98 will merge it in the next time she reviews PRs. Great job!

**Pull request merged**
